modules/gdscript/doc_classes/GDScriptSyntaxHighlighter.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="GDScriptSyntaxHighlighter" inherits="EditorSyntaxHighlighter" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
+	<brief_description>
+		A GDScript syntax highlighter that can be used with [TextEdit] and [CodeEdit] nodes
+	</brief_description>
+	<description>
+		[b]Note:[/b] This class can only be used for editor plugins because it relies on editor settings.
+		[codeblocks]
+		[gdscript]
+		var code_preview = TextEdit.new()
+		var highlighter = GDScriptSyntaxHighlighter.new()
+		code_preview.syntax_highlighter = highlighter
+		[/gdscript]
+		[/codeblocks]
+	</description>
+	<tutorials>
+	</tutorials>
```

Comment:
No need to change, I just noticed that for some reason we require an empty `<tutorials></tutorials>` even though for the other member types we remove empty groups.